File: VB/Q245342/MainForm.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.XtraEditors

Namespace Q245342
	Partial Public Class MainForm
		Inherits XtraForm
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub MainForm_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
			dataTable1.Rows.Add(0D)
		End Sub
	End Class
End Namespace

File: CS/Q245342/MainForm.cs
using System;
using DevExpress.XtraEditors;

namespace Q245342 {
    public partial class MainForm :XtraForm {
        public MainForm() {
            InitializeComponent();
        }

        private void MainForm_Load(object sender, EventArgs e) {
            dataTable1.Rows.Add(0m);
        }
    }
}
